README.md

```diff
@@ -43,3 +56,7 @@ Create a file named `.env` to place configuration options in.
 * SERVER_DB_CONNECTION_STRING
 * SERVER_START_DATE
 * SERVER_END_DATE
+* SERVER_STRIPE_SECRET
```

Comment:
I don't think we should use the stripe API key as our bearer token for the webhook endpoint